### ISSUE TITLE
RPC Error: Unexpected Content-Type text/plain

### ISSUE THREAD

**Issue by sotarules** [lightninglabs/loop#213]
Friends every weekend I update LND and Loop (go) to their latest versions.  Loop has been working well for me, although I haven't tried it for a while.

Today I tried to loop in and I received this:
```sh
root@router:~# loop in 1200000
[loop] rpc error: code = Unknown desc = Internal Server Error: HTTP status code 500; transport: received the unexpected content-type "text/plain; charset=utf-8"
root@router:~#
```
Here is my loop console:
```sh
root@router:/etc/supervisor/conf.d# /root/go/bin/loopd
2020-06-01 15:43:54.688 [INF] LOOPD: Version: 0.2.4-alpha commit=
2020-06-01 15:43:54.689 [INF] LNDC: Creating lnd connection to localhost:10009
2020-06-01 15:43:54.689 [INF] LNDC: Connected to lnd
2020-06-01 15:44:12.880 [INF] LNDC: Using network mainnet
2020-06-01 15:44:12.880 [INF] LOOPD: Swap server address: swap.lightning.today:11009
2020-06-01 15:44:12.885 [INF] STORE: Checking for schema update: latest_version=1, db_version=1
2020-06-01 15:44:12.887 [INF] LOOPD: Starting gRPC listener
2020-06-01 15:44:12.889 [INF] LOOPD: Starting REST proxy listener
2020-06-01 15:44:12.890 [INF] LOOPD: Starting swap client
2020-06-01 15:44:12.890 [INF] LOOPD: Waiting for updates
2020-06-01 15:44:12.891 [INF] LOOPD: RPC server listening on 127.0.0.1:11010
2020-06-01 15:44:12.891 [INF] LOOPD: REST proxy listening on 127.0.0.1:8081
2020-06-01 15:44:21.765 [INF] LOOP: Connected to lnd node [redacted] with pubkey [redacted]
2020-06-01 15:44:21.766 [INF] LOOP: Wait for first block ntfn
2020-06-01 15:44:21.768 [INF] LOOP: Starting event loop at height 632659
2020-06-01 15:48:24.670 [INF] LOOPD: Loop in quote request received
```
I've tried downgrading loop and that didn't seem to fix it. I cannot find any other his on the internet.  Any idea about what to try next?

Thanks in advance for your help.

**Comment by Roasbeef** [comment]
Thanks for this report! I checked on things on our end, and something was out of place configuration wise, things should be working again for _that version of `loopd`. As you're running a pretty old version (latest is 6.0+), I'd recommend you update, as the latest version has a ton of bug fixes and new features such as multi-loop support.

**Comment by sotarules** [comment]
Just to note I had a problem with my update script that made the binary never update, I fixed that and am now running the 0.6.2-beta and it works fine.
